src/renderer/reducers/swap.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import { handleActions } from "redux-actions";
+import type { AvailableProviderV3 } from "@ledgerhq/live-common/lib/exchange/swap/types";
+
+export type SwapStateType = {
+  providers: ?Array<AvailableProviderV3>,
+  pairs: ?$PropertyType<AvailableProviderV3, "pairs">,
+};
+
+const initialState: SwapStateType = { providers: null, pairs: null };
```

Comment:
there is a new reducers added, is it stored in app.json?

-> the risk of shipping this as "silently non impacting" is you must not change this format anymore or support "migrating from previous format", because all users, even if feature is not visible, will start to have this family and present with this value.

Reply:
It's not a persisted state @gre it's used within the form only and not persisted to data
cc @jgiamporcaro-ledger